### DISCUSSION TITLE
How can I show full image

### DISCUSSION THREAD

**Discussion by parsagholipour** [igordanchenko/yet-another-react-lightbox#239] [General]
I have the issue of losing some parts of the image on different sizes. How can I force it to show the full image?
![image](https://github.com/igordanchenko/yet-another-react-lightbox/assets/57809920/716c1ea8-a3ef-4187-91b1-5f8539acc586)

Here it shows more:
![image](https://github.com/igordanchenko/yet-another-react-lightbox/assets/57809920/6d4c5627-ad37-481e-bb45-3860e01cb1dd)

On different screen sizes it changes.

**Comment by igordanchenko** [comment]
I'm afraid there isn't much I can do to troubleshoot your issue without seeing your actual code. Please feel free to fork the following sandbox and create a reproducible example. 

https://stackblitz.com/edit/yet-another-react-lightbox-sandbox?file=src%2FApp.tsx
